projects/step-core/src/lib/services/bookmark.service.ts

```diff
@@ -0,0 +1,69 @@
+import { inject, Inject, Injectable } from '@angular/core';
+import { BehaviorSubject, map, Observable, of } from 'rxjs';
+import { StorageProxy } from '../modules/basics/shared/storage-proxy';
+import { LOCAL_STORAGE } from '../modules/basics/shared/storage.token';
+import { Bookmark } from '../shared/bookmark';
+import { BookmarkStorageService } from './bookmark-storage.service';
+
+const bookmark = 'BOOKMARKS';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class BookmarkService extends StorageProxy {
+  private _bookmarkStorageService = inject(BookmarkStorageService);
+  private bookmarksInternal$ = new BehaviorSubject<Bookmark[]>(this._bookmarkStorageService.getStorageBookmarks());
+  readonly bookmarks$ = this.bookmarksInternal$.asObservable();
+  readonly bookmarkMenuEntries$ = this.bookmarks$.pipe(
+    map((bookmarks) =>
+      bookmarks.map((element) => {
+        const menuEntry = {
+          title: element.label!,
+          id: element.link!,
+          icon: element.icon!,
+          parentId: 'bookmarks-root',
+          weight: 1000 + bookmarks.length,
+          isEnabledFct(): boolean {
+            return true;
+          },
+        };
+        return menuEntry;
+      }),
+    ),
+  );
+
+  constructor(@Inject(LOCAL_STORAGE) storage: Storage) {
+    super(storage, bookmark);
+    this.bookmarksInternal$.next(this._bookmarkStorageService.getStorageBookmarks());
```

Comment:
This invocation also redundant, as the same action is done in field's initialization

Reply:
Removed whole constructor.